src/app/components/MediaLoader/configs/aresMedia.ts

```diff
@@ -53,6 +59,10 @@ export default ({
 
   const versionID = versionsBlock?.versionId ?? '';
 
+  const orientation =
```

Comment:
Just thinking should use we the `embedded` flag here as well to always set it to `landscape` when `embedded === true`?

`embedded` is only `true` when the MediaLoader is being rendered via the av-embeds routes, so I feel like we should always render the player in 16/9 and then let the iframe do the sizing if needed.

Reply:
Yeah that makes sense, didn't really think about the embedded version, it will look weird if we don't do this, will add it
<img width="996" alt="image" src="https://github.com/user-attachments/assets/3bb259ef-b3c0-4db5-8b5a-79da2bb3fff1">

Reply:
Cheers 🙏 

In hindsight, the MediaLoader as a compound component may be the way to go in the long term so that av-embeds can have its own version so don't have to worry about adding bits and pieces.

Reply:
I've added the logic and a test, good spot!